node/src/components/consensus/era_supervisor.rs

```diff
@@ -59,8 +59,7 @@ use crate::{
 
 /// The unbonding period, in number of eras. After this many eras, a former validator is allowed to
 /// withdraw their stake, so their signature can't be trusted anymore.
-// TODO: This needs to be in sync with AUCTION_DELAY/booking_duration_millis. (Already duplicated!)
-const BONDED_ERAS: u64 = 4;
+const BONDED_ERAS: u64 = DEFAULT_UNBONDING_DELAY - AUCTION_DELAY;
```

Comment:
Why are we subtracting `AUCTION_DELAY`? Let's assume that validator became bonded in era-10 and immediately sends a `withdraw` request. According to the `DEFAULT_UNBONDING_PERIOD`, it will be able to withdraw its stake in era-24 but consensus will consider it unbonded in era-20 already. 

Am I missing something?

Reply:
See [this comment](https://github.com/CasperLabs/casper-node/pull/360#discussion_r504099334).